tests/UnitTests/Rsp.IrasService.UnitTests/Web/Controllers/ReviewBodyControllersTests/DisableReviewBodyTests.cs

```diff
@@ -35,28 +29,17 @@ public async Task Disable_ShouldSendCommand(Guid id)
     }
 
     [Theory, AutoData]
-    public async Task Disable_ShouldSendCommand_AndLogAuditTrail_WhenReviewBodyIsDisabled(Guid id,
-        ReviewBodyDto reviewBodyDto, IEnumerable<ReviewBodyAuditTrailDto> reviewBodyAuditTrailDtos)
+    public async Task Disable_ShouldSendCommand_WhenReviewBodyIsDisabled(Guid id,
+        ReviewBodyDto reviewBodyDto)
```

Comment:
If method signature or parameters names are longer, move them to new line, other keep them on a single line for 1 or 2 params
```suggestion
    public async Task Disable_ShouldSendCommand_WhenReviewBodyIsDisabled(Guid id, ReviewBodyDto reviewBodyDto)
```
or

```suggestion
    public async Task Disable_ShouldSendCommand_WhenReviewBodyIsDisabled
    (
        Guid id,
        ReviewBodyDto reviewBodyDto
    )
```